count(*),fault_log_entry_id
5,13
